packages/cli/test/integration.js

```diff
@@ -2683,7 +2683,7 @@ test('deploy a Lambda with 3 seconds of maxDuration', async t => {
   const url = new URL(output.stdout);
 
   // Should time out
-  url.pathname = '/api/wait-for/4';
+  url.pathname = '/api/wait-for/5';
```

Comment:
any reason this couldn't be 100000?

Reply:
Because that would make CI slower, no need to wait much more than 3 seconds

Reply:
I guess I was assuming that this would timeout after 3 seconds no matter how long the route actually takes.

Reply:
Oh good point. Thats probably why this assertion wasn't flaky, it was the other one